lib/book.py

```diff
@@ -292,9 +285,13 @@ def send_xray(self, device_books, overwrite=True, already_created=True, log=None
                         
                     if original_asin and original_asin != new_asin:
                         # if we changed the asin, update the image file name
-                        thumbname_orig = os.path.join(device, "system", "thumbnails", "thumbnail_%s_EBOK_portrait.jpg" % original_asin)
+                        device_root = os.path.join(info['device_book'].split(os.sep)[0], os.sep)
```

Comment:
I'll test it once you've pushed it to master - more faff otherwise to have to manually pull/install.